src/cmd/singularity/inspect_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/sylabs/singularity/src/pkg/test"
+)
+
+func inspectImage(image string) ([]byte, error) {
+	args := []string{"inspect", image}
+	cmd := exec.Command(cmdPath, args...)
+
+	return cmd.CombinedOutput()
+}
+
+func TestInspect(t *testing.T) {
+	var definition = "../../../examples/busybox/Singularity"
+	var image = "Default.sif"
+
+	if b, err := buildImage(definition, image); err != nil {
```

Comment:
Looks like this function doesn't exist

Reply:
it was rm on https://github.com/sylabs/singularity/pull/2128 , to fix #2127
we should all use  `imageBuild()`
```go
	opts := buildOpts{
		force:    true,
		sandbox:  false,
		writable: false,
	}
	if b, err := imageBuild(opts, imagePath, definition); err != nil {
		t.Log(string(b))
		t.Fatalf("unexpected failure: %v", err)
	}
	imageVerify(t, imagePath, true)
```